# Disable some warnings that become errors due to other patches
# Should be removed after warnings caused by other patches are fixed

--- a/build/common.gypi
+++ b/build/common.gypi
@@ -5046,6 +5046,7 @@
           'WARNING_CFLAGS': [
             '-Wall',
             '-Wextra',
+            '-Wno-unused',
             # Don't warn about unused function parameters.
             '-Wno-unused-parameter',
             # Don't warn about the "struct foo f = {0};" initialization
